martin/src/srv/tiles_info.rs

```diff
@@ -9,7 +9,8 @@ use serde::Deserialize;
 use tilejson::{tilejson, TileJSON};
 
 use crate::source::{Source, TileSources};
-
+use crate::srv::SrvConfig;
+use crate::utils::parse_base_path;
```

Comment:
please run `just fmt2` - there should be an empty line after `use` section. I wonder why `cargo fmt` didn't do it?

Reply:
Would run fmt2 after the nightly toolchain is installed(might take 30 minutes as my internet connection is poor...)

Reply:
hm, i think we should change fmt2 to use the stable version of cargo, but use the "unstable feature" of it.